packages/myst-execute/tests/run.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, test, expect } from 'vitest';
+import fs from 'node:fs';
+import path from 'node:path';
+import yaml from 'js-yaml';
+import { kernelExecutionTransform, launchJupyterServer, ICache } from '../src';
+import type { GenericParent } from 'myst-common';
+import { VFile } from 'vfile';
+import { KernelManager, ServerConnection, SessionManager } from '@jupyterlab/services';
+import { default as nodeFetch } from 'node-fetch';
+import { Headers, Request, Response } from 'node-fetch';
+import type { IExpressionResult } from 'myst-common';
+import type { IOutput } from '@jupyterlab/nbformat';
+
+// fetch polyfill for node<18
+if (!globalThis.fetch) {
+  globalThis.fetch = nodeFetch as any;
+  globalThis.Headers = Headers as any;
+  globalThis.Request = Request as any;
+  globalThis.Response = Response as any;
```

Comment:
Was this supposed to be in the CLI as well eventually to patch `fetch`?

Reply:
It is :)